more/free-programming-cheatsheets.md

```diff
@@ -13,6 +13,7 @@
 * [JavaScript](#javascript)
   * [jQuery](#jquery)
   * [Vue.js](#vuejs)
+  * [React.js](#reactjs)
```

Comment:
```suggestion
  * [React.js](#reactjs)
  * [Vue.js](#vuejs)
```

The new category was not in alphabetical order.

Reply:
Looks good now thanks a lot.